roles/ara_frontend_nginx/defaults/main.yaml

```diff
@@ -0,0 +1,21 @@
+---
+#  Copyright (c) 2019 Red Hat, Inc.
+#
+#  This file is part of ARA Records Ansible.
+#
+#  ARA Records Ansible is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  ARA Records Ansible is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with ARA Records Ansible. If not, see <http://www.gnu.org/licenses/>.
```

Comment:
```suggestion
---
# Copyright (c) 2021 The ARA Records Ansible authors
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

:point_up_2: is shorter, inclusive of Red Hat but also other authors -- I haven't got around to updating it everywhere but it's what I have been using for new files (like https://github.com/ansible-community/ara/blob/master/ara/ui/templatetags/diff_result.py#L1-L2)

Reply:
Ah, cool, whenever I do change one of these, I'll try not to forget this ;-)